src/confdb/confdb_setup.h

```diff
@@ -22,6 +22,11 @@
 #ifndef CONFDB_SETUP_H_
 #define CONFDB_SETUP_H_
 
+#include <talloc.h>
+
+#include "confdb_private.h"
```

Comment:
Why?
Forward declaration should be enough for `confdb_ctx *`.

Reply:
Forward declarations are quite annoying to debug.

Reply:
Why/how?

I guess there should be a reason for "_private" in the header name, probably it's not intended to be included outside confdb module...

Reply:
Although I tend to disagree with those "_private" files, I will respect the structure already implemented.